ScanCommand/src/main/java/io/ballerina/scan/ScanCmd.java

```diff
@@ -0,0 +1,133 @@
+ *  * Version 2.0 (the "License"); you may not use this file except
+ *  * in compliance with the License.
+ *  * You may obtain a copy of the License at
+ *  *
+ *  *   http://www.apache.org/licenses/LICENSE-2.0
+ *  *
+ *  * Unless required by applicable law or agreed to in writing,
+ *  * software distributed under the License is distributed on an
+ *  * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  * KIND, either express or implied. See the License for the
+ *  * specific language governing permissions and limitations
+ *  * under the License.
+ *
+ */
+
+package io.ballerina.scan;
+
+import io.ballerina.cli.BLauncherCmd;
+import io.ballerina.scan.utilities.StringToListConverter;
+import picocli.CommandLine;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.PrintStream;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.List;
+
+@CommandLine.Command(name = "scan", description = "Perform static code analysis for ballerina packages")
+public class ScanCmd implements BLauncherCmd {
+
+    private final PrintStream outputStream;
+    @CommandLine.Parameters(description = "Program arguments")
+    private final List<String> argList = new ArrayList<>();
+    @CommandLine.Option(names = {"--help", "-h", "?"}, hidden = true)
+    private boolean helpFlag;
+
+    @CommandLine.Option(names = {"--platform-triggered"}, hidden = true)
+    private boolean platformTriggered;
+
+    @CommandLine.Option(names = "--target-dir", description = "Target directory path")
+    private String targetDir;
+
+    @CommandLine.Option(names = {"--scan-report"}, description = "Enable scan report generation")
+    private boolean scanReport;
+
+    @CommandLine.Option(names = {"--list-rules"},
+            description = "List the internal rules available in the Ballerina scan tool.")
+    private boolean listRules;
+
+    @CommandLine.Option(names = {"--include-rules"},
+            converter = StringToListConverter.class,
+            description = "Specify the comma separated list of rules to include specific analysis issues")
+    private List<String> includeRules = new ArrayList<>();
+
+    @CommandLine.Option(names = {"--exclude-rules"},
+            converter = StringToListConverter.class,
+            description = "Specify the comma separated list of rules to exclude specific analysis issues")
+    private List<String> excludeRules = new ArrayList<>();
+
+    @CommandLine.Option(names = {"--platforms"},
+            converter = StringToListConverter.class,
+            description = "static code analysis output platform",
+            defaultValue = "local")
+    private List<String> platforms = new ArrayList<>();
+
+    public ScanCmd() {
+        this.outputStream = System.out;
+    }
+
+    @Override
+    public String getName() {
+        return "scan";
```

Comment:
How about we make "scan" a constant and use it in the `@CommandLine.Command->name` field and here in the `getName()` method?